app/views/schools/add_participants/yourself.html.erb

```diff
@@ -0,0 +1,23 @@
+<% content_for :title, "Add yourself as mentor" %>
+
+<% content_for :before_content, govuk_back_link(text: "Back", href: back_link_path) %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <span class="govuk-caption-xl"><%= @school.name %> - <%= @cohort.display_name %> cohort</span>
+    <h1 class="govuk-heading-xl">Are you sure you want to add yourself as a mentor?</h1>
+
+    <p class="govuk-body">The induction tutor and mentor roles are separate. <%= govuk_link_to "Check what each role needs to do", roles_schools_cohort_path, class: "govuk-link--no-visited-state" %>.</p>
+
+    <p class="govuk-body">You should only assign yourself as a mentor in exceptional circumstances. If your school needs you to do this, you should:</p>
+
+    <ul class="govuk-list govuk-list--bullet">
+      <li>make sure you can take on the full responsibilities of both roles</li>
+      <li>put safeguards in place so that your role as an ECT’s mentor is kept separate from your role in their assessment</li>
+    </ul>
+
+    <%= form_for add_participant_form, url: { action: :complete }, method: :post do |f| %>
+      <%= f.govuk_submit "Confirm" %>
+    <% end %>
```

Comment:
This could probably be replaced with `govuk_button_to`, but this is also fine

Reply:
Agreed! Now updated